helix-core/src/selection.rs

```diff
@@ -122,7 +122,7 @@ impl Range {
 
     #[inline]
     pub fn fragment<'a, 'b: 'a>(&'a self, text: RopeSlice<'b>) -> Cow<'b, str> {
-        Cow::from(text.slice(self.from()..self.to() + 1))
+        Cow::from(text.slice(self.from()..self.to()))
```

Comment:
This needs to be end inclusive or it won't include the character the cursor is on. Probably need a safe wrapper to check if `to + 1` is valid though

Reply:
I changed it to not need it as I removed the subtraction in `helix-term`/src/commands.rs:
`let max_to = doc.text().len_chars().saturating_sub(1);` to
`let max_to = doc.text().len_chars();`